be/src/exec/rowid_fetcher.cpp

```diff
@@ -136,10 +138,16 @@ Status RowIDFetcher::_merge_rpc_results(const PMultiGetRequest& request,
             if (output_block->is_empty_column()) {
                 *output_block = vectorized::Block(_fetch_option.desc->slots(), 1);
             }
+            if (serdes.empty() && col_uid_to_idx.empty()) {
```

Comment:
is possible here serde is not empty but col_uid_to_idx empty ?

Reply:
that is impossible